src/patternfly/components/FormControl/form-control.scss

```diff
@@ -149,4 +149,12 @@ $pf-c-form-control__select--ViewBox: "320" !default;
       --pf-c-form-control--invalid--Background: var(--pf-c-form-control__select--invalid--Background);
     }
   }
+
+  &.pf-m-resize-vertical {
+    resize: vertical;
+  }
```

Comment:
Maybe a `min-height` here?

Reply:
@mceledonia what about a `min-height` when you can resize it vertically to prevent sizing it down too short?